notes/introduction.md

```diff
@@ -0,0 +1,61 @@
+## Introduction
+
+This section is non-normative.
+
+A central goal of Solid and of the Web is to "empower an equitable, informed and interconnected society" [ETHICAL-WEB].  In order to work together online towards these goals people and communities need a voice - a way to describe who they are; a discovery mechanism - a way to find the resources they need; and an identifying mechanism - a way to be found and trusted for purposes of collaboration. A person or organization's `Solid Profile` is a set of statements which fulfill or point to resources which fulfill one or more of those functions - identification and description of the profile owner, discovery of their resources and the resources of their social network. A Solid Profile is thus an entryway to a person or organization's presence in the "vast inter-twingled tapestry" [NELSON] that is the collaborative and empowering web.
+
+### Identification
```

Comment:
These sections (Identifiication, Description, Discovery, etc), I feel, are getting into too much detail and stepping into comformance/requirements. I would keep them as a mention in the introduction but wouldn't have subsections of the introduction, as these are discussed in the text itself.